README.md

```diff
@@ -216,8 +216,18 @@ psql# create user test;
 # Set password for the user
 psql# alter user test with password 'test';
 ```
-</br>
-</br>
+
+#### Test coverage
+
+To correctly calculate test coverage, you need to run the coverage with the `--concurrency=thread,gevent` parameter:
+
+```bash
+coverage run --concurrency=thread,gevent -m pytest
```

Comment:
can it be async for concurrency?

Reply:
This is needed to analyze asynchronous code. Details of the problem are described here:
https://github.com/nedbat/coveragepy/issues/1082